plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -51,8 +51,8 @@ function init(name) {
   replaceFileVariables(packagePath);
   replaceFileVariables(indexPath);
 
-  // @todo: this is to make testing locally easier. Remove before merge.
-  const proc1 = child_process.spawnSync('npm', ['link', '@kinvolk/headlamp-plugin'], {cwd: dstFolder});
+  // This can be used to make testing locally easier.
+  // const proc1 = child_process.spawnSync('npm', ['link', '@kinvolk/headlamp-plugin'], {cwd: dstFolder});
```

Comment:
Don't we just have to run `npm link` in the `headlamp-plugin` folder? If that's all, I think we should remove this commented out code.